azure-pipelines-pr.yml

```diff
@@ -0,0 +1,90 @@
+variables:
+  # Cannot use key:value syntax in root defined variables
+  - name: _TeamName
+    value: DotNetCore
+  - name: _PublishUsingPipelines
+    value: true
```

Comment:
Is there any value in this _PublishUsingPipelines variable?  if it's only used once, and never changes, we should just move it into the yaml where it's used so we don't have to bounce around to understand the value.